File: C/1019.c
#include <stdio.h>
 
int main() {
 
    int aux;
    scanf("%d", &aux);
    int hour = (aux/3600);
    aux %= 3600;
    int minutes = (aux/60);
    aux %= 60;
    printf("%d:%d:%d\n", hour,minutes,aux);
 
    return 0;
}

File: Javascript/1019.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var aux = parseInt(lines.shift());
var hour = aux/3600;
aux %= 3600;
console.log("%d:%d:%d", parseInt(hour),parseInt(aux/60),parseInt(aux%60));